src/shared/ui/components/table/index.tsx

```diff
@@ -0,0 +1,416 @@
+                                            {headerString}
+                                        </Typography>
+
+                                        <Typography
+                                            sx={{ color: (theme) => theme.palette.text.secondary }}
+                                            component="div"
+                                        >
+                                            {data && data[index]}
+                                        </Typography>
+
+                                        {collapsible && index + 1 === header.length && (
+                                            <IconButton
+                                                aria-label="expand row"
+                                                size="small"
+                                                // onClick={() => setIsOpen(open => !open)}
+                                                sx={{ flexGrow: 0 }}
+                                            >
+                                                <KeyboardArrowUpIcon />
+                                                {/* {isOpen ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />} */}
+                                            </IconButton>
+                                        )}
+                                    </CardRow>
+                                ))}
+                            </Box>
+                        </Card>
+                    ))}
+                </>
+            ) : (
+                <StyledTableContainer>
+                    <table>
+                        <StyledTableHead>
+                            <TableRow>
+                                {header.map((string, index) => (
+                                    <TableCell key={index}>
+                                        {sortHeaders &&
+                                        sortHeaders.includes(string) &&
+                                        order &&
+                                        orderBy &&
+                                        createSortfunction ? (
+                                            <TableSortLabel
+                                                active={orderBy === string}
+                                                direction={orderBy === string ? order : "asc"}
+                                                onClick={createSortfunction(string)}
+                                                IconComponent={sortIcon}
+                                                sx={{
+                                                    "& .MuiTableSortLabel-icon": {
+                                                        transform: "none",
+                                                        opacity: 1,
+                                                    },
+
+                                                    "& .MuiTableSortLabel-iconDirectionDesc .sort-down": {
+                                                        fill: (theme) =>
```

Comment:
for css like this lets find a different solutions, we are not even using an mui theme